content/4_0_document_oriented.tex

```diff
@@ -0,0 +1,47 @@
+\begin{figure}
+    \includegraphics[width=\textwidth]{img/dbRanked.png}
+    \caption{Ranking of documented oriented databases}
+    \label{figure:Ranking}
+\end{figure}
+Overall not many changes happened in the ranking over that last three years. The development and establishment of these databases can therefore be seen as a longer running process. Three years is a short time frame to retrace the whole market development. To demonstrate how the top 5 as well as RethinkDB (currently 11) established themselves over a longer period consider \autoref{figure:Development}. It provides insight in the development of the document oriented database market and the popularity of the leading five products in ranking since 2013 \parencite{dbEngineRankingPopularity}. 
+
+\begin{figure}
+    \centering
+    \includegraphics[width=\textwidth]{img/dbRankedTrend.png}
+    \caption{Development of the document oriented database market}
+    \label{figure:Development}
+\end{figure}
+
+Ever since 2013 MongoDB has had a leading role. However, the top 3 of today were not as set back then. Further \autoref{figure:Development} finally provides a bit more insight over the development of several products over time. CouchDB especially, once second to MongoDB, moved down to rank 5, while Amazon DynamoDB made a name for itself and claimed that second rank. Even though RethinkDB was recently overtaken by Microsoft Azure Cosmos DB it has an astonishing increase in popularity considering its small beginning back in 2013. So in the long run things weren't as set as they have been over the last three years. The historic development of the products and especially the features added, which lead to a sometimes rather steep ascend in popularity, are relevant still.
+
+\subsection{Comparison relational database management systems (RDMS)}
+\label{section:postalcodes}
+Before going into details of specific document store databases this chapter will focus on comparing classical relational database management systems (RDMS) with the new concept of document store databases. This will provide a closer insight into the structure and working of document store databases as well as its general advantages and disadvantages not only in direct comparison.\\
```

Comment:
Please use clear lines instead of line breaks. In my opinion, paragraphs increase readability of text.

Reply:
Same as in \newpage.